### ISSUE TITLE
install error: file INSTALL cannot find libhpx_parcel_coalescing.so.0.9.11

### ISSUE THREAD

**Issue by Syntaf** [STEllAR-GROUP/hpx#1842]
I'm getting an error when attempting to install the release branch, tested on both my local machine and hermione, same exact error.

The specific error is:
```
Install the project...
-- Install configuration: "Release"
CMake Error at cmake_install.cmake:48 (FILE):
  file INSTALL cannot find
  "/home/gdev-home/Github/hpx/build/lib/hpx/libhpx_parcel_coalescing.so.0.9.11".
```

###### steps to reproduce:
 --------

* Clone the release branch
* build HPX with g++ 
* make 
* make install

build script used on my local machine (xfce ubuntu 14.04 LTS)
```
#!/bin/bash

REPO=$1

QUICKBOOK=/usr/bin
AUTOINDEX=/usr/bin
XSLTPROC=/usr/bin
DOCBOOK_DTD=/usr/share/xml/docbook/schema/dtd/4.2
DOCBOOK_XSL=/usr/share/xml/docbook/stylesheet/docbook-xsl
BOOST_ROOT=/usr/local/include/boost
HWLOC_ROOT=/usr/local/include/hwloc

if [[ -n "$REPO" ]]; then
    cmake -DBOOSTQUICKBOOK_ROOT=$QUICKBOOK             \
          -DBOOSTAUTOINDEX_ROOT=$AUTOINDEX                    \
          -DXSLTPROC_ROOT=$XSLTPROC                           \
          -DDOCBOOK_DTD_ROOT=$DOCBOOK_DTD                     \
          -DDOCBOOK_XSL_ROOT=$DOCBOOK_XSL                     \
          -DHPX_WITH_DOCUMENTATION=On                         \
          -DBOOST_ROOT=$BOOST_ROOT                            \
          -DHWLOC_ROOT=$HWLOC_ROOT                            \
          -DCMAKE_INSTALL_PREFIX=/home/gdev-home/Github/hpx/build \
          $REPO
fi
```

build script used on hermione 
```
#!/bin/bash                                                                      
                                                                                 
                                                                                 
SCRIPTPATH=$( cd $(dirname $0) ; pwd -P )                                        
INSTALL_PATH=$SCRIPTPATH/install                                                 
BUILD_PATH=$SCRIPTPATH/build                                                     
                                                                                 
if [ "$1" = "clean" ]; then                                                      
    rm -rf $INSTALL_PATH $BUILD_PATH                                             
    exit 0                                                                       
fi                                                                               
                                                                                 
CC=/usr/bin/                                                                     
                                                                                 
if [ ! -d "$BUILD_PATH" ]; then                                                  
  mkdir -p "$BUILD_PATH"                                                         
                                                                                 
  cd "$BUILD_PATH"                                                               
                                                                                 
  cmake -DCMAKE_INSTALL_PREFIX=$HOME/hpx/build               \                   
        -DBOOST_ROOT=$HOME/boost/install                     \                   
        -DCMAKE_BUILD_TYPE=Release                           \                   
        -DHPX_HAVE_PARCELPORT_MPI=True                       \                   
        -DHPX_MALLOC="jemalloc"                              \                   
        -Wdev                                                \                   
        $SCRIPTPATH/repo                                                         
                                                                                 
fi                                                                               
                                                                                 
cd "$BUILD_PATH"                                                                 
                                                                                 
make $1 -k -j24      
```

**Comment by sithhell** [comment]
I can not reproduce this. Buildbot also seems to be happy. For example:
http://hermione.cct.lsu.edu/builders/hpx_gcc_4_8_boost_1_53_debian_x86_64_release/builds/742/steps/install/logs/stdio

**Comment by hkaiser** [comment]
Is this the same as #1529?

**Comment by Syntaf** [comment]
@hkaiser I believe so. 

There may be something wrong with my fork if it can't be reproduced, i'll try building a direct clone of the branch when I get home

**Comment by Syntaf** [comment]
Installing works fine on master, but strangely it fails if I try to install on a fork with no changes to the codebase.

**Comment by Syntaf** [comment]
Closed, since this is the same as #1529
